### ISSUE TITLE
ebert.zip broken

### ISSUE THREAD

**Issue by qiuhaolu** [NPoe/ebert#1]
Hello, 
Thanks for sharing the code.
I downloaded the pretrained model file ebert.zip, but it seems broken and cannot be decompressed. Would you fix it?

**Comment by kenanfa3** [comment]
Yes! I'm having the same issue I can't access pre-trained wiki2vec model "ebert.zip"

**Comment by NPoe** [comment]
Hi, thanks for pointing this out. It seems to have been a disk quota issue on my end.

I have removed E-BERT-large for now. Can you check the updated README and see if you can download E-BERT-base? I will add back the large model once I have figured out where to host it.

**Comment by qiuhaolu** [comment]
> Hi, thanks for pointing this out. It seems to have been a disk quota issue on my end.
> 
> I have removed E-BERT-large for now. Can you check the updated README and see if you can download E-BERT-base? I will add back the large model once I have figured out where to host it.

Thanks for fixing it. I have downloaded them and will test them later. 

Is the wikipedia2vec-base-cased the E-BERT-base, which is wikipedia2vec after linear mapping to BERT, or is it just the pre-trained wikipedia2vec embeddings?

And what is the .npy file, is it the trained mapper, i.e., the output of run_mapping.py? Sry I'm a bit confused and cannot tell from the file names.

**Comment by NPoe** [comment]
> > Hi, thanks for pointing this out. It seems to have been a disk quota issue on my end.
> > I have removed E-BERT-large for now. Can you check the updated README and see if you can download E-BERT-base? I will add back the large model once I have figured out where to host it.
> 
> Thanks for fixing it. I have downloaded them and will test them later.
> 
> Is the wikipedia2vec-base-cased the E-BERT-base, which is wikipedia2vec after linear mapping to BERT, or is it just the pre-trained wikipedia2vec embeddings?
> 
> And what is the .npy file, is it the trained mapper, i.e., the output of run_mapping.py? Sry I'm a bit confused and cannot tell from the file names.

wikipedia2vec-base-cased is just pre-trained Wikipedia2Vec. The *linear.npy file is the linear mapper that turns Wikipedia2Vec vectors into E-BERT vectors. You can recreate this mapper with run_mapping.pay.

**Comment by drukermanOG** [comment]
Hello,

Attempting once again to access the mapper but it seems that the server isn't available. Is it down? or perhaps there's a new host and relevant link?
